### DISCUSSION TITLE
Is it possible for WingetUI to use winget already present in the system as a part of AppInstaller?

### DISCUSSION THREAD

**Discussion by hexerus** [Devolutions/UniGetUI#1481] [Q&A] [answered]
Hi,

It seems that currently WingetUI picks up `winget` from `C:\Program Files\WingetUI\PackageManagers\winget-cli_x64` folder. But is it possible for the app to use the one already present in system and added to path through AppInstaller? The one that can be accessed anywhere by using `winget` command in CMD or Powershell. If I remove the winget-cli folder, the UI app does not detect winget as present in the system.

**Comment by marticliment** [comment, marked as answer]
You can enable using System Winget from WingetUI Settings -> Experimental features.

However, this option might not work as expected if your system language is not english
